### PR TITLE
anystyle-cli: Init at 1.3.10

### PR DESCRIPTION
**Pull request by SCOTT-HAMILTON**
###### Motivation for this change
Answer to https://github.com/NixOS/nixpkgs/issues/96082 packaging request.

###### Things done
Packaged anystyle-cli 1.3.10 with bundix (had to modify the gemspec file though).

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by matthuszagh** [comment]
Thanks for packaging this @SCOTT-HAMILTON! Only comment is that I needed `poppler_utils` additionally to get this to work. Not sure if that should also be a dep. Otherwise looks good.

**Comment by doronbehar** [comment]
> Thanks for packaging this @SCOTT-HAMILTON! Only comment is that I needed `poppler_utils` additionally to get this to work. Not sure if that should also be a dep. Otherwise looks good.

@SCOTT-HAMILTON then it might be a good idea to wrap the executable like this:

```nix
  preFixup = ''
    wrapProgram --prefix PATH ${poppler_utils}/bin $out/bin/anystyle
  '';
```

Or similarly.

File: pkgs/tools/misc/anystyle-cli/Gemfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+group :profile do
+	gem 'ruby-prof', require: false
+	gem 'gnuplot', require: false
+end
+
+group :extra do
+	gem 'lmdb'
+  gem 'redis'
+  gem 'redis-namespace'
+  gem 'yard'
```
Comment (doronbehar) [inline pkgs/tools/misc/anystyle-cli/Gemfile:36]: Mixed indentation causing EditorConfig issues:

https://github.com/NixOS/nixpkgs/pull/96186/checks?check_run_id=1652060791

File: pkgs/tools/misc/anystyle-cli/default.nix [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        sha256 = "1w79zcia60nnnyrmyvpd10pmxrpk5c7lj9gmmblhwi8x5mfq9k0n";
+        type = "gem";
+      };
+    };
+  };
+in
+buildRubyGem rec {
+
+  inherit ruby;
+  name = "${gemName}-${version}";
```
Comment (doronbehar) [inline pkgs/tools/misc/anystyle-cli/default.nix:26]: I'm not experienced at all with `buildRubyGem`, is it possible this is redundent? Or at least could be replaced with `pname = gemName;`?

Comment (SCOTT-HAMILTON) [reply pkgs/tools/misc/anystyle-cli/default.nix]: You're right, fixed.

File: pkgs/tools/misc/anystyle-cli/Gemfile (hunk 1 above)
Comment (SCOTT-HAMILTON) [reply pkgs/tools/misc/anystyle-cli/Gemfile]: Fixed.

**Review by taku0 — APPROVED** [review]
Works fine.

**Review by taku0 — APPROVED** [review]

**Pull request merged**